src/skimage/registration/_affine.py

```
@@ -0,0 +1,1180 @@
+            im0, im1, bins=30, weights=w
+        ),
+        minimizer="Powell",
+        max_iter=100,
+        max_trial=10,
+        min_trial=4,
+        options={
+            "maxfev": 10000,
+            "disp": False,
+            "xtol": 1e-6,
+            "ftol": 1e-3,
+        },
```

Ruff (0.15.0)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)